pv.js

```diff
@@ -37,6 +39,9 @@ class Pv {
                          }[setting];
         this.CLPV =      { 'PHOENIXGO':  (/\([^()]*\)/g) }[setting];
     }
+    fetchOgspvType() {
```

Comment:
Is this function used? Why does this pr add code?

Reply:
fixed, same as https://github.com/online-go/gtp2ogs/pull/260#discussion_r425345678